app/views/layouts/application.html.erb

```diff
@@ -21,6 +21,12 @@
         <%= link_to 'Users Dashboard', admin_users_path unless current_user.default? %>
         <%= link_to 'NPCs Dashboard', admin_npcs_path unless current_user.default? %>
         <%= link_to 'Log Out', logout_path %>
+        <form action="https://www.paypal.com/donate" method="post" target="_top">
+        <input type="hidden" name="hosted_button_id" value="C2KF65M3S2VLJ" />
+        <input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donate_LG.gif" border="0" name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate with PayPal button" />
+        <img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1" />
+        </form>
+
```

Comment:
I need to find a way to test for this officially, but just using it on local, it does work.